0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -155,7 +153,9 @@ The mapping from the ILP address to the local ledger address is dependent on the
 #### getBalance
 <code>ledgerPlugin.getBalance() ⇒ Promise.&lt;String></code>
 
-Return an integer string representing the current balance. Plugin must be connected, otherwise the promise should reject.
+Return an integer decimal string (`..., '-3', '-2', '-1', '0', '1', '2', '3', ...`) representing the current balance, in the ledger's base unit. For example, on a ledger with `currencyCode` 'USD' and `currencyScale` 6,
```

Comment:
ok, will change it to "(base-ten)" to avoid confusion with the "digit behind the decimal point" meaning of the word "decimal"